### ISSUE TITLE
Crash when parsing RegularExpressionLiteral PropertyName

### ISSUE THREAD

**Issue by ziyunfei** [anba/es6draft#7]
```
js> ({/ /(){}})
java.lang.AssertionError
	at com.github.anba.es6draft.parser.TokenStream.peekToken(TokenStream.java:372)
	at com.github.anba.es6draft.parser.Parser.SAFE_LOOKAHEAD(Parser.java:1810)
	at com.github.anba.es6draft.parser.Parser.propertyDefinition(Parser.java:7558)
	at com.github.anba.es6draft.parser.Parser.objectLiteral(Parser.java:7386)
	at com.github.anba.es6draft.parser.Parser.primaryExpression(Parser.java:6921)
	at com.github.anba.es6draft.parser.Parser.leftHandSideExpression(Parser.java:7986)
	at com.github.anba.es6draft.parser.Parser.unaryExpression(Parser.java:8242)
	at com.github.anba.es6draft.parser.Parser.binaryExpression(Parser.java:8343)
	at com.github.anba.es6draft.parser.Parser.assignmentExpression(Parser.java:8507)
	at com.github.anba.es6draft.parser.Parser.assignmentExpressionNoValidation(Parser.java:8466)
	at com.github.anba.es6draft.parser.Parser.coverParenthesizedExpressionAndArrowParameterList(Parser.java:7001)
	at com.github.anba.es6draft.parser.Parser.primaryExpression(Parser.java:6932)
	at com.github.anba.es6draft.parser.Parser.leftHandSideExpression(Parser.java:7986)
	at com.github.anba.es6draft.parser.Parser.unaryExpression(Parser.java:8242)
	at com.github.anba.es6draft.parser.Parser.binaryExpression(Parser.java:8343)
	at com.github.anba.es6draft.parser.Parser.assignmentExpression(Parser.java:8507)
	at com.github.anba.es6draft.parser.Parser.assignmentExpression(Parser.java:8451)
	at com.github.anba.es6draft.parser.Parser.expression(Parser.java:8922)
	at com.github.anba.es6draft.parser.Parser.expressionStatement(Parser.java:5585)
	at com.github.anba.es6draft.parser.Parser.statement(Parser.java:4933)
	at ...(Frames omitted:13)

**Comment by anba** [comment]
Thanks for reporting! I'll fix the bug later this week.

**Comment by anba** [comment]
Fixed in b90b28b12127b3c4f429be02f05a24ff66298b3e
